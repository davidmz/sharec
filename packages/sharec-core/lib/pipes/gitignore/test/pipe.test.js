const { fixtures } = require('testUtils')
const { pipe } = require('../pipe')

describe('strategies > pipes > gitignore', () => {
  const gitignoreBaseFxt = fixtures('gitignore/lines/00-base')

  it('handles .gitignore files', () => {
    expect(pipe('.gitignore')(gitignoreBaseFxt)).toMatchSnapshot()
  })
})
