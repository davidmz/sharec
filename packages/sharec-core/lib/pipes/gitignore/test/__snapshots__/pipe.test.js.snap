// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`strategies > pipes > gitignore handles .gitignore files 1`] = `
"node_modules
foo
bar
baz"
`;
